assistive_gym/envs/utils/smpl_geom.py

```diff
@@ -75,7 +75,7 @@ def generate_geom(model_path, smpl_data = None):
     hull_dict = {}
 
     # create joint geometries
-    geom_dir = "/home/louis/Documents/Projects/assistive-gym/assistive_gym/envs/assets/human/meshes/"
+    geom_dir = "/home/hrl5/Documents/Projects/assistive-gym/assistive_gym/envs/assets/human/meshes/"
```

Comment:
I think you could just dont commit this change. I will fix it later to make it directory agnostic